lizmap/lizmap_api/config.py

```diff
@@ -205,6 +205,9 @@ def __init__(self, project, fix_json=False):
             'hideProject': {
                 'wType': 'checkbox', 'type': 'boolean', 'default': False
             },
+            'wmsSingleRequestForAllLayers': {
```

Comment:
Sorry I forgot, all new keys are lower snake case, so it should `wms_single_request_for_all_layers`.

It's not obvious in this area of the code, it was legacy.